models/tmp/stg_reddit_ads__account_report_tmp.sql

```diff
@@ -1 +1,4 @@
-select * from {{ var('account_report') }}
+{{ config(enabled=var('ad_reporting__reddit_ads_enabled', True)) }}
```

Comment:
Similar comment from the staging model. This will need to be added to all the tmp models so they may be disabled if an Ad Reporting user needs to disable them.

Reply:
added to all the tmp files